content/docs/release-notes.md

```diff
@@ -2,7 +2,54 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Safekeeper: back up WAL to S3 for disaster recovery.
+
+* Safekeeper: download WAL from S3 on demand.
+
+* Safekeeper: replace callmemaybe with etcd subscriptions on safekeeper timeline info.
```

Comment:
I suggest to put `callmemaybe` in accents and explicitly call out what it is. call? internal api call? internal api call from safekeeper to pageserver?

Reply:
changed to "`callmemaybe` communication API between safekeepers and pageserver"